textMining.py

```diff
@@ -0,0 +1,111 @@
+import wikipedia
```

Comment:
Header comment with basic information about this code (author's name, brief file description, etc) would be good.